src/map-view.ios.ts

```diff
@@ -214,6 +214,7 @@ class MapViewDelegateImpl extends NSObject implements GMSMapViewDelegate {
 
     public didTapMyLocationButtonForMapView(mapView: GMSMapView): boolean {
         const owner = this._owner.get();
+        owner.setDelegate()
```

Comment:
tried running it locally, but this method doesn't seem to exist:
```
map-view.ios.ts:217:15 - error TS2551: Property 'setDelegate' does not exist on type 'MapView'. Did you mean '_delegate'?
```

Reply:
Hi,

Tes I forged to remove It, I’ve tried to implement 2 delegate in order to activate the button myLocation on ios...

Reply:
@Burgov, Fixed here [43b51e5](https://github.com/dapriett/nativescript-google-maps-sdk/pull/440/commits/43b51e5fd9b237e2ed99431892835b841f0b4348)

Did you try it ?